ipamutils/utils.go

```diff
@@ -35,18 +35,37 @@ type NetworkToSplit struct {
 }
 
 // InitNetworks initializes the broad network pool and the granular network pool
-func InitNetworks(defaultAddressPool []*NetworkToSplit) {
+func InitNetworks(defaultAddressPool []*NetworkToSplit, isGlobalScope bool) {
```

Comment:
does not seem a clean solution this one.
I would create a Reinit function if needed